### PR TITLE
Added ShiftRightClick to inventory click attributes

### PR DESCRIPTION
**Pull request by ozi2285**
Just Added New Click Attribute to inventory click

### CODE REVIEW

File: MinecraftClient/CommandHandler/ArgumentType/InventoryActionArgumentType.cs [hunk 1]
```diff
@@ -1,47 +1,49 @@
...
+                "left" => WindowActionType.LeftClick,
+                "leftclick" => WindowActionType.LeftClick,
+                "right" => WindowActionType.RightClick,
+                "rightclick" => WindowActionType.RightClick,
+                "mid" => WindowActionType.MiddleClick,
+                "middle" => WindowActionType.MiddleClick,
+                "middleclick" => WindowActionType.MiddleClick,
+                "shift" => WindowActionType.ShiftClick,
+                "shiftclick" => WindowActionType.ShiftClick,
+                "shiftrightclick" => WindowActionType.ShiftRightClick,
```
Comment (ozi2285) [inline MinecraftClient/CommandHandler/ArgumentType/InventoryActionArgumentType.cs:37]: the only part added here.

File: MinecraftClient/Inventory/WindowActionType.cs [hunk 2]
```diff
@@ -1,83 +1,88 @@
...
+
+        /// <summary>
+        /// Shift+Left click with mouse on a slot: send a whole item stack to the hotbar or other inventory
+        /// </summary>
+        ShiftClick,
+
+        /// <summary>
+        /// Shift+Right click with mouse on a slot: send a whole item stack to the hotbar or other inventory
+        /// </summary>
+        ShiftRightClick,
```
Comment (ozi2285) [inline MinecraftClient/Inventory/WindowActionType.cs:31]: only added one here is this. it does same thing with shift click only difference is what reported to server(which when needed it's useful).

File: MinecraftClient/Protocol/Handlers/Protocol18.cs [hunk 3]
```diff
@@ -3723,6 +3723,11 @@ public bool SendHeldItemChange(short slot)
                         mode = 1;
                         item = new Item(ItemType.Null, 0, null);
                         break;
+                    case WindowActionType.ShiftRightClick:
+                        button = 1;
```
Comment (ozi2285) [inline MinecraftClient/Protocol/Handlers/Protocol18.cs:3727]: copy paste shiftclick and change button variable to 1 and it's done.

File: MinecraftClient/McClient.cs [hunk 4]
```diff
@@ -2125,6 +2125,550 @@ public bool DoWindowAction(int windowId, int slotId, WindowActionType action)
                             }
                         }
                         break;
+                    case WindowActionType.ShiftRightClick:
```
Comment (ozi2285) [inline MinecraftClient/McClient.cs:2128]: code under this is complete same code under shiftclick it's just same like i said. only difference of them what reported to server and when it's needed by some plugin to do action it works.

**Comment by ReinforceZwei** [comment]
Hi, I've updated the master branch to fix line endings issue. Please update your forked repository. Also, please change all line endings to LF (currently you are using CRLF). After that, I can start reviewing your PR.

**Pull request closed without merging**